src/components/fin/metricSlider/TabNavigation.tsx

```diff
@@ -0,0 +1,87 @@
+'use client'
+
+import Trans from '@/components/translation/Trans'
+import Badge from '@/design-system/layout/Badge'
+import { useClientTranslation } from '@/hooks/useClientTranslation'
+import { useCurrentMetric } from '@/hooks/useCurrentMetric'
+import { twMerge } from 'tailwind-merge'
+import HeadingButtons from './heading/HeadingButtons'
+
+type Props = {
+  isSticky?: boolean
+  isStatic?: boolean
+  shouldShowWater?: boolean
+}
+export default function TabNavigation({
+  isSticky,
+  isStatic,
+  shouldShowWater,
+}: Props) {
+  const { t } = useClientTranslation()
+
+  const { currentMetric, setCurrentMetric } = useCurrentMetric()
+
+  return (
+    <div
+      className={twMerge(
+        'pointer-events-auto relative flex w-full items-end justify-between bg-white transition-all',
+        isSticky ? 'pt-3' : 'pt-0.5'
+      )}>
+      <div className="flex">
+        <button
+          aria-label={t('Mon empreinte carbone')}
+          onClick={() => setCurrentMetric('carbone')}
+          className={twMerge(
+            'z-40 mb-0 rounded-t-xl border-2 px-4 pb-1 pt-2 text-lg font-medium transition-all duration-300',
+            currentMetric !== 'carbone'
```

Comment:
2 smalls remarks
 - Lot of use of `'carbone'` and `'eau'` magic strings in some places. You should use more constants from `'@/constants/metric'` 
 
 The file should be
 
 ```typescript
 import { Metric } from '@/publicodes-state/types'

export const carboneMetric = 'carbone' as const

export const eauMetric = 'eau' as const

export const metrics: Metric[] = [carboneMetric, eauMetric]

export const defaultMetric: Metric = carboneMetric
 ```
 
 - Use a map instead of a ternary in every places where you test `currentMetric`
 
 Example here
 
 Before
 ```typescript
 currentMetric !== 'carbone'
              ? ' border-transparent border-b-primary-50 text-primary-700'
              : 'border-x-primary-50 border-b-transparent border-t-primary-50 bg-gray-100'
 ```
 after
 
 ```typescript
import { carboneMetric, eauMetric } from '@/constants/metric'

const currentMetricButtonClasses: Record<Metric, string> = {
   [carboneMetric]: 'border-transparent border-b-primary-50 text-primary-700',
   [eauMetric]: 'border-x-primary-50 border-b-transparent border-t-primary-50 bg-gray-100',
}

        <button
          className={twMerge(
            'z-40 mb-0 rounded-t-xl border-2 px-4 pb-1 pt-2 text-lg font-medium transition-all duration-300',
            currentMetricButtonClasses[currentMetric]
          )}></button>
```